examples/hydro/reynolds_num_example.py

```diff
@@ -0,0 +1,13 @@
+from symplyphysics.laws.hydro import reynolds_number
+from symplyphysics import units, convert_to, dimensionless
+
+rho = 1000 * units.kilogram / units.meter**3
```

Comment:
Quantities are not defined this way. See other examples for reference.
Also example is not required for the test task.